src/main/kotlin/no/nav/syfo/dialogmelding/converter/DokumentDialogmeldingConverter.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.syfo.dialogmelding.converter
+
+import no.kith.xmlstds.msghead._2006_05_24.ObjectFactory
+import no.kith.xmlstds.msghead._2006_05_24.XMLDocument
+import no.nav.syfo.behandler.domain.BehandlerDialogmeldingBestilling
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+class DokumentDialogmeldingConverter(melding: BehandlerDialogmeldingBestilling) {
+    private val dialogmeldingConverter: DialogmeldingConverter
+    private var document: XMLDocument? = null
+    fun getDocument(): XMLDocument {
+        ensureDocument()
+        return document!!
+    }
+
+    private fun ensureDocument() {
```

Comment:
Jeg skjønner ikke helt denne koden hvor når verdi settes til null, også står det etterpå at det aldri skal være null når den brukes.

Hvis en verdi aldri skal være null, så burde den aldri settes til null i utgangspunktet.

Reply:
Lage en PR på refaktorering av konvertings-filene for eksisterende oppfolgingsplan-xml her:
https://github.com/navikt/isdialogmelding/pull/47

Reply:
Ja, det stemmer at denne koden tar utgangspunkt i det som brukes for oppfølgingsplaner.